src/Raven.Studio/typings/_studio/dto.d.ts

```diff
@@ -1003,6 +1003,12 @@ interface ReactDirtyFlag {
     setIsDirty: (isDirty: boolean, customDialog?: () => JQueryPromise<confirmDialogResult>) => void;
 }
 
+interface ReactProps<PathParams = any, QueryParams = Record<string, unknown>> {
+    pathParams?: PathParams;
+    queryParams?: QueryParams;
+    location?: databaseLocationSpecifier
+}
+
```

Comment:
Let's spit it into 2 types - ReactPropsWithPathParams and ReactPropsWithQueryParams. Thanks to that we dont need to pass any every time.

e.g.

```
interface ReactPropsWithQueryParams<T extends Record<string, any>> extends ReactProps {
    queryParams?: T;
}
```